### ISSUE TITLE
question regarding comment in rbl.conf

### ISSUE THREAD

**Issue by heraklit256** [rspamd/rspamd#3116]
Just a quick question since I stumbled across this when upgrading to rspamd 2.0: What is the privacy concern here?

As far as I am concerned, the FQDN in question is already passed to Spamhaus via their DBL, so passing the single IP addresses resolved from that FQDN to their RBL does not give them any new information.

Please correct me if I am wrong. :-)

**Comment by heraklit256** [comment]
See https://github.com/rspamd/rspamd/blob/master/conf/modules.d/rbl.conf#L274 for the comment.

**Comment by vstakhov** [comment]
It gives *a lot of* new information to those domains' owners (e.g. spammers or cybercriminals).

**Comment by vstakhov** [comment]
And it is aside of the performance conciderations where resolve_ip sucks and can suck even more if attackers are clever enough...

**Comment by heraklit256** [comment]
@vstakhov: Thank you for this explanation, now I've got it.

I agree with you on your point that resolving a FQDN leaks information to its owner, the nameserver operators (if they differ), and much people in between. Talking about performance, this is much slower indeed.

My intention was just to understand this since I observed a lot of spam containing unlisted FQDNs which resolve to IP addresses listed at SBL or XBL. So, with rspamd 2.0, I will just add the lines commented out to my local configuration.
